### DISCUSSION TITLE
ABCD_site variable

### DISCUSSION THREAD

**Discussion by sanaashrafali** [now-i-know-my-abcd/docs#8] [Stats & Code Help] [answered]
Advice on locating an abcd_site variable on the NDA data dictionary?

**Comment by sanaashrafali** [comment, marked as answer]
The variable site_id_l is in the NDA data dictionary in the ABCD Longitudinal Tracking (short name "abcd_lt01").

**Comment by fhaist** [comment]
In addition, the site_id_l to site name code is included in the release notes for the Longitudinal Tracking instrument. That can be found in _12. NDA 4.0 Other Non-Imaging Instruments.docx_
